src/gmail.js

```diff
@@ -2561,6 +2567,31 @@ var Gmail = function(localJQuery) {
             return flag_value === "0" || flag_value === undefined;
         }
     };
+    
+	//To know button labels setting is set to text (or icon)
+    api.check.is_buttensLabel_text = function() {
+        if( api.check.is_new_data_layer() ) {
+            var flag = 0;
+            
+            if(document.querySelector("div[gh='tm'] div[class='Bn']") !== null) {
+                flag = 1;
+            }
+            
+            return flag;
+        } else {	//To handle classic gmail UI           	
```

Comment:
We want this function to return a bool (`true` or `false`), not `0` or `1`, right?

So why not just write `return document.querySelector("div[gh='tm'] div[class='Bn']") !== null;` ?